website/docs/dbt-cli/install/pip.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: "pip"
+title: "Use pip to install dbt"
+description: "You can use pip to install dbt Core and adapter plugins from the command line."
 ---
 
 dbt Core and plugins are Python modules distributed on [PyPi](https://pypi.org/project/dbt/), and can be installed via `pip`. We recommend using virtual environments when installing with `pip`.
```

Comment:
@jtcohen6 Can you provide a bit about why someone would use pip over Homebrew? Also would you "install from source" to get the absolute latest code? And, finally, why would someone use Docker over the others?

Reply:
Great questions!
- Homebrew only works on Mac, so `pip` is required on Windows, Linux, etc
- Installing from source lets people get the latest code, or to install dbt from a specific commit. In practice, this is relevant for contributing changes, or debugging a past change
- Docker is most useful in production, since it comes with dbt and all of its dependencies already installed (no risk of `pip install dbt-snowflake` failing in the middle of the night because of some borked dependency. To a lesser extent, some folks might use it for local development, if their machine doesn't have a python environment set up, but I'd say that's a lot less common